scripts/publish-prepare.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -ev
+npm run clean
```

Comment:
I think we should also remove node_modules here and install stuff again to make sure that everything is clear.